### PR TITLE
chore: update README.rst for XAI get_metadata. 

### PR DESCRIPTION
**Pull request by ji-yaqi**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -316,6 +316,31 @@ To delete an endpoint:
   endpoint.delete()
 
 
+XAI: Get_metadata
```
Comment (sasha-gitg) [inline README.rst:319]:
```suggestion
Explainable AI: Get Metadata
```

File: README.rst [hunk 2]
```diff
@@ -316,6 +316,31 @@ To delete an endpoint:
...
+-----------------
+
+To get metadata from tf1 models:
```
Comment (sasha-gitg) [inline README.rst:322]:
```suggestion
To get metadata from TensorFlow 1 models:
```

File: README.rst [hunk 3]
```diff
@@ -316,6 +316,31 @@ To delete an endpoint:
...
+.. code-block:: Python
+
+  from google.cloud.aiplatform.explain.metadata.tf.v1 import saved_model_metadata_builder
+  
+  builder = saved_model_metadata_builder.SavedModelMetadataBuilder(
+            'gs://python/to/my/model/dir', tags=[tf.saved_model.tag_constants.SERVING]
+        )
+  generated_md = builder.get_metadata()
+
+To get metadata from tf2 models:
```
Comment (sasha-gitg) [inline README.rst:333]:
```suggestion
To get metadata from TensorFlow 2 models:
```

File: README.rst (hunk 2 above)
Comment (sasha-gitg) [reply README.rst:322]: Preference to lead with TF 2 ahead of TF 1 as I think it will be more common.

**Review by sasha-gitg — APPROVED** [review]

**Review by vinnysenthil — APPROVED** [review]

**Comment by ji-yaqi** [comment]
Seems like we have some failures for samples: https://source.cloud.google.com/results/invocations/e04134ba-d96b-42ec-a3dc-34b169075155/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-aiplatform%2Fsamples%2Flint%2Fpresubmit/log. Are they expected?

**Pull request merged**